assets/src/design-system/components/typography/list/index.js

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import styled, { css } from 'styled-components';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import { THEME_CONSTANTS, themeHelpers } from '../../../theme';
+import { defaultTypographyStyle } from '../styles';
+
+export const List = styled.ul(
```

Comment:
Adding in a list component to the design system because this has come up on both new iterations of the checklist and it feels like if we have other lists in the editor or dashboard we should make them consistent.